tests/unit/s2n_signature_scheme_test.c

```diff
@@ -50,9 +52,49 @@ int main(int argc, char **argv)
                         sig_prefs->signature_schemes[dup_i];
                 EXPECT_NOT_EQUAL(sig_scheme->iana_value, potential_duplicate->iana_value);
             }
+
+            /* Must be included in s2n_signature_preferences_all */
+            bool in_all = false;
+            for (size_t all_i = 0; all_i < all_prefs->count; all_i++) {
+                if (sig_scheme == all_prefs->signature_schemes[all_i]) {
+                    in_all = true;
+                }
+            }
+            EXPECT_TRUE(in_all);
         }
         policy_i++;
     }
 
+    /* Test: s2n_signature_preferences_all should also include s2n_rsa_pkcs1_md5_sha1
+     *
+     * s2n_rsa_pkcs1_md5_sha1 is the implicit default for pre-TLS1.2 when no signature
```

Comment:
Do we need a test to make sure is not in any security policy? Except the test_all policy of course.

Reply:
I would say we don't need one, but I can add one anyway :P